envctl/env_scripts/python/compute.sh

```diff
@@ -28,13 +28,13 @@ destroy() {
   export GCR_PATH=gcr.io/$PROJECT_ID/logging:$SERVICE_NAME
   gcloud container images delete $GCR_PATH -q --force-delete-tags 2> /dev/null
   # delete service
-  gcloud compute instances delete $SERVICE_NAME -q
+  gcloud compute instances delete $SERVICE_NAME --zone us-west1-c -q
   set -e
 }
 
 verify() {
   set +e
-  gcloud compute instances describe $SERVICE_NAME > /dev/null 2> /dev/null
+  gcloud compute instances describe $SERVICE_NAME --zone us-west1-c > /dev/null 2> /dev/null
```

Comment:
Is pinning to specific zone required? I just wonder if we could have capacity issues...

Reply:
This is just for our tests, which run for maybe 30 minutes a day. Trying to split each test between different zones would make clean-up much more complicated, without much benefit